src/d3d12/d3d12_staging_buffer.cpp

```diff
@@ -12,13 +12,13 @@ namespace wr::d3d12
 		buffer->m_target_resource_state = resource_state;
 		buffer->m_size = size;
 		buffer->m_stride_in_bytes = stride;
-		buffer->m_is_staged = false;
+		buffer->m_is_staged = true;
```

Comment:
Sure about this change?

Reply:
Yeah, to my knowledge this isn't used anywhere, and was just something so a buffer wouldn't get staged twice. But this isn't necessary anymore after staging buffer rewrites.